client/templates/profile/profile.html

```diff
@@ -0,0 +1,83 @@
+<template name="profile">
+<div id="profile">
+  <section class="profile-top">
+    <div class="container">
+      <div class="row">
+        {{#if editMode}}
+          {{#with userInfo}}
+            {{> profileEdit}}
+          {{/with}}
+        {{else}}
+          <div class="col-md-6 profile-top-left">
+            {{#with userInfo}}
+              <div class="col-md-3">
+                <img src="{{#if profile.avatar.image_192 }} {{profile.avatar.image_192}} {{else}} {{profile.avatar.default}} {{/if}}" class="img-circle" alt="profile image"/>
+              </div>
+              <div class="col-md-9">
+                <h3>{{username}}
+                  {{#if isCurrentUserProfile currentUser}}
+                    <a href="#profileEdit" class="editProfile"><i class="fa fa-pencil" aria-hidden="true"></i>{{_ "Edit Profile"}}</a>
+                  {{else}}
+                    {{#if status.online}}
+                      <a><i class="fa fa-circle online-now" aria-hidden="true"></i></a>
+                    {{/if}}
+                  {{/if}}
+                </h3>
+                <p>
+                  <i class="fa fa-map-marker" aria-hidden="true"></i>{{profile.location}}
+                </p>
+                <div class="social-icons">
+                  <a href="https://twitter.com/{{profile.social.twitter}}" target="_blank">
+                    <i class="fa fa-twitter" aria-hidden="true"></i>
+                  </a>
+                  <a href="https://www.facebook.com/{{profile.social.facebook}}" target="_blank">
+                    <i class="fa fa-facebook" aria-hidden="true"></i>
+                  </a>
+                  <a href="https://github.com/{{profile.social.github}}" target="_blank">
+                    <i class="fa fa-github" aria-hidden="true"></i>
+                  </a>
+                  <a href="https://www.linkedin.com/in/{{profile.social.linkedin}}" target="_blank">
+                    <i class="fa fa-linkedin" aria-hidden="true"></i>
+                  </a>
+                </div>
+              </div>
+            {{/with}}
+          </div>
+          <div class="col-md-6 profile-top-right">
+            <p class="bio">{{userInfo.profile.bio}}</p>
+            <a href="{{userInfo.profile.website}}" target="_blank">
+              {{userInfo.profile.website}}
```

Comment:
can you move `website` with other social elements, and use either `globe` or `link` icon for it ? no need to modify it, in `schema`.